prac_07/project.py

```diff
@@ -0,0 +1,9 @@
+
+
+
+class project:
+    def __init__(self, Name, Start_Date_Priority, Cost_Estimate, Completion_Percentage):
+        self.Name = Name
+        self.Start_Date_Priority = Start_Date_Priority
+        self.Cost_Estimate = Cost_Estimate
+        self.Completion_Percentage = int(Completion_Percentage)
```

Comment:
I don't think object creation checks variable type, that is the job of the main program using the class.
Also, percentage may be a float value.